etc/pickup-sample-app/pickup-oidc/src/main/webapp/home.jsp

```diff
@@ -295,7 +295,7 @@
                                 <%
                                     if (claimsSet != null) {
                                         Map<String, Object> hashmap = new HashMap<>();
-                                        hashmap = claimsSet.getCustomClaims();
```

Comment:
what is the difference between `.getCustomClaims` and `getClaims`? any reason for removing ``.getCustomClaims`?

Reply:
getCustomClaims method is no longer available as it was in the ReadOnlyJWTClaimsSet class. It was returning a hashmap of key value pair claims which are not predefined. 
The JWTClaimsSet class has the getClaims method which also returns the key value claims as a hashmap.